### PR TITLE
examples: Add doc on how to use the SSHD example

### LINKED ISSUE #1674: Q: How do you connect to the SSHD example?
I built the "moby" CLI and then ran `build/run` on the SSHD example on my Mac. I can see the system booted, but it's not clear how to connect to it. Is that even possible?

https://github.com/linuxkit/linuxkit/blob/master/examples/sshd.yml

### PR DESCRIPTION
**Pull request by rneugeba**
Signed-off-by: Rolf Neugebauer <rolf.neugebauer@docker.com>

resolves #1674

### CODE REVIEW

**Comment by rneugeba** [comment]
ping @alexellis

**Comment by alexellis** [comment]
Thanks @rneugeba

File: examples/sshd.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# SSH example
+
+The LinuKit [sshd example](./sshd.yaml) defines an image running a SSH
```
Comment (alexellis) [inline examples/sshd.md]: LinuKit -> LinuxLit

 sshd example was is broken for me.

**Comment by alexellis** [comment]
I've just tried it out, worked great. Thanks 👍

**Comment by rneugeba** [comment]
thanks for pointing out the type. Pushed fix

**Pull request merged**